### DISCUSSION TITLE
Welcome to caniwebview.com Discussions!

### DISCUSSION THREAD

**Discussion by NiklasMerz** [WebView-CG/Compatibility-Data-Project#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. 

**This forum could become  a place where WebView developers meet and can connect**

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by niutech** [comment]
Hello! I am a current maintainer of https://html5test.co. If you are interested in contributing to it, please join me at https://github.com/niutech/html5test.
